extensions/sleep-timers/package.json

```
@@ -0,0 +1,127 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "sleep-timers",
+  "title": "Sleep Timers",
```

```suggestion
  "name": "sleep-timer",
  "title": "Sleep Timer",
```

And rename the folder too